.github/workflows/release.yaml

```diff
@@ -277,10 +277,14 @@ jobs:
         run: |
           for container in trust trust-ui trust-docs; do
             IMAGE="ghcr.io/${{ github.repository_owner }}/${container}:${{ needs.init.outputs.version }}"
+            LATEST="ghcr.io/${{ github.repository_owner }}/${container}:latest"
```

Comment:
This can be a bit dangerous, as any tag will be pushed as `latest`, independent of the ordering of versions.